.gitlab-ci.yml

```diff
@@ -142,8 +139,6 @@ test-performance:
     - $DNF_WITH_OPTIONS install $RPM_REQUIREMENTS
     - python3.9 -m pip install tox
     - tox -e corgi -- -m performance --no-cov
-  rules:
```

Comment:
I'd rather keep the performance test as a scheduled task as it doesn't test the current branch, but the main branch itself. It's annoying to have the CI/CD fail for something not related to your current changes.